Makefile

```diff
@@ -373,3 +373,13 @@ release-push-%:
 	$(MAKE) clean-bin
 	$(MAKE) ARCH=$* build
 	$(MAKE) ARCH=$* push
+
+# SvCat Kubectl plugin stuff
+############################
+svcat: .init .generate_files \
+		$(BINDIR)/svcat/svcat
```

Comment:
Why into it's own directory?

Reply:
Because I was just copy pasting it from the makefile I wrote for my plugins, which needed their own dirs. Will fix.